airflow/dags/ingestion/ask-astro-load.py

```diff
@@ -48,8 +48,10 @@
     "airflow",
 ]
 
+schedule_interval = "@daily" if ask_astro_env == "prod" else None
```

Comment:
Should we also have a sample `.env` file for airflow deployment @pankajastro ?

Reply:
sometimes people keeps secret also in `.env` file so I feel it safe to not check in